app/assets/stylesheets/show.scss

```diff
@@ -1,6 +1,8 @@
 .show-container {
   margin: 10px;
   padding: 5px;
+  background-color: $lighter_blue;
```

Comment:
Name of variable `lighter_blue` should be written in all lowercase letters with hyphens instead of underscores